integration_tests/cdk/requirements.txt

```diff
@@ -0,0 +1,15 @@
+aws-cdk-lib>=2.99.1
+aws_cdk.aws_cognito_identitypool_alpha>=2.99.0a0
+aws-cdk.aws-apigatewayv2-alpha>=2.99.0a0
+constructs>=10.0.0,<11.0.0
```

Comment:
@emileten some requirements are not really needed because they are defined in eoapi-cdk 

https://github.com/developmentseed/eoapi-cdk/blob/1e39ddb54597754d4d621313e7ce34eebe9a8a6f/package.json#L53-L56

Reply:
Some of these, we don't use them explicitly in `integration_tests/cdk`, in which case I agree I should remove them. 

However shouldn't we still include those we _explicitly_ use, such as `constructs`: https://github.com/developmentseed/eoapi-cdk/blob/b4de0358eadd3cdabae3076ea4b8189b572d7bcd/integration_tests/cdk/eoapi_template/pgStacInfra.py#L6.